app/views/logix/index.html.erb

```diff
@@ -1,3 +1,12 @@
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+<script src="js/scripts.js"></script>
+
+<script>
+  $(document).ready(function(){
+    $("#alertdiv").delay(10000).fadeOut(600);
```

Comment:
Great job @BBroadbent ! 
The delay before the fadeOut seems a little too much, could we reduce it to something like 5s?